contact_info_service.py

```
@@ -0,0 +1,141 @@
+                    "address",
+                    "city",
+                    "state",
+                    "website_or_email",
+                    "phone",
+                    "source_urls",
+                ],
+                "additionalProperties": False,
+            },
+            "strict": True,
+        }
+    }
```

Ruff (0.14.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)